From da7f94fdc9b8ac5a1e5837d17e3802a3af66d613 Mon Sep 17 00:00:00 2001
From: Szehon Ho <szehon@cloudera.com>
Date: Fri, 13 Mar 2015 15:03:05 -0700
Subject: [PATCH 0090/1164] CDH-26107: Fix Test overridden_confs

---
 .../hive/ql/hooks/VerifyOverriddenConfigsHook.java |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/itests/util/src/main/java/org/apache/hadoop/hive/ql/hooks/VerifyOverriddenConfigsHook.java b/itests/util/src/main/java/org/apache/hadoop/hive/ql/hooks/VerifyOverriddenConfigsHook.java
index aa01baf..57baef0 100644
--- a/itests/util/src/main/java/org/apache/hadoop/hive/ql/hooks/VerifyOverriddenConfigsHook.java
+++ b/itests/util/src/main/java/org/apache/hadoop/hive/ql/hooks/VerifyOverriddenConfigsHook.java
@@ -38,7 +38,7 @@
   // a config variable not in the default List of config variables, and a config variable in the
   // default list of config variables, but which has not been overridden
   private static String[] keysArray =
-    {"hive.exec.post.hooks", "hive.config.doesnt.exit",
+    {"mapred.job.tracker", "hive.exec.post.hooks", "some.hive.config.doesnt.exit",
      "hive.exec.mode.local.auto"};
   private static List<String> keysList = Arrays.asList(keysArray);
 
-- 
1.7.9.5

